analysis/statistics/routing_cost.py

```diff
@@ -0,0 +1,11 @@
+from typing import Tuple
+
+from heuristic.classes import Solution
+
+
+def routing_cost(solution: Solution) -> Tuple[str, float]:
+    """
+    Returns Solution routing costs.
```

Comment:
No need to reference the class here - it's just _solution_, as in, this is a normal English text. (here and in other places)